lib/generators/access_lint/install/install_generator.rb

```diff
@@ -0,0 +1,29 @@
+module AccessLint
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      desc "Add AccessLint routes and include statements."
+      def insert_route
+        insert_into_file "config/routes.rb", :after => "Rails.application.routes.draw do\n" do
+          <<-RUBY
+  if Rails.env.test? || Rails.env.development?
+    mount AccessLint::Rails::Engine, at: "access_lint"
+  end
```

Comment:
Try `String#strip_heredoc` here to avoid the awkward alignment? http://apidock.com/rails/String/strip_heredoc